doc/compodoc_sources/how-to-guides/create-entity-details-panel.md

```diff
@@ -0,0 +1,47 @@
+# How to create an Entity Details Panel
```

Comment:
terminology: Should we call these "panels" (but actually that's the tabs that can contain multiple of these components), sub-components (super abstract and bulky term ...) or what's a good name?

Reply:
That's a good way to think about it :+1: 

A panel can hold multiple components, which are then called "panel components"

Reply:
```suggestion
# How to create an Entity Details Panel Component
```